docs/olm.md

```diff
@@ -8,15 +8,27 @@ Ensure the Hive namespace exists:
 oc new-project hive
 ```
 
-Identify the current version of the operator you wish to replace, if applicable. (TODO)
+Identify the current version of the operator you wish to replace, if applicable. If not, you should be able to just specify a fake version.
 
 ```
 GIT_HASH=`git rev-parse --short HEAD`
 GIT_COMMIT_COUNT=`git rev-list 9c56c62c6d0180c27e1cc9cf195f4bbfd7a617dd..HEAD --count`
 ./hack/generate-operator-bundle.py bundle/ 0.1.5-4b53b492 $GIT_COMMIT_COUNT $GIT_HASH quay.io/dgoodwin/hive:latest alpha
```

Comment:
Make it obvious that quay.io/dgoodwin/hive:latest should be replaced with your own image, OR use the default latest image from CI.